src/main/java/net/minecraftforge/fml/client/FMLClientHandler.java

```diff
@@ -594,7 +594,7 @@ public IResourcePack getResourcePackFor(String modId)
     @Override
     public String getCurrentLanguage()
     {
-        return client.getLanguageManager().getCurrentLanguage().getLanguageCode();
+        return (client.getLangaugeManager().getCurrentLanguage() == null) ? "en_US" : client.getLanguageManager().getCurrentLanguage().getLanguageCode();
```

Comment:
Perhaps store the value of the method call in a variable and then check against the variable so it doesn't have to be called multiple times?